SAVE_DESIRED_POS: 0
SAVE_IMAGES: 1

# The camera intrinsic parameters matrix
# 3x3 matrix
camera_intrinsic_parameters: [550, 0,    376.0,
                              0,    550, 240.0,
                              0,    0,    1]

# Sampling time
dt: 0.01

# Gain on linear velocities
# gainv: .85
gainv: .85

# Gain on angular velocities
gainw: 1

# Threshold on the feature position error, used to stop the visual control
feature_error_threshold: .008
# feature_error_threshold: .0025

nfeatures: 500
scaleFactor: 1.2
nlevels: 8
edgeThreshold: 15
patchSize: 30
fastThreshold: 20
flann_ratio: 0.7

############################################################
# Cambio de control 1 o 2.
# 1 Es para 1/dist
# 2 Es para dist
control: 1

############################################################
# Cambio de cámara 0, 1 o 2.
# 0 Para cámara de humminbird
# 1 Para cámara de iris - frontal
# 2 Para cámara de iris - abajo
# 3 Para cámara de iris - Ambas
camara: 1